.jenkinsci/bindings.groovy

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env groovy
+
+def doBindings() {
+
+    def cmake_options = ""
+    if (params.JavaBindings) {
+        cmake_options += " -DSWIG_JAVA=ON "
+    }
+    if (params.PythonBindings) {
+        cmake_options += " -DSWIG_PYTHON=ON "
+    }
+    // In case language specific options were not set,
+    // build for every language
+    if (!params.JavaBindings && !params.PythonBindings) {
+        cmake_options += " -DSWIG_JAVA=ON -DSWIG_PYTHON=ON "
+    }
+    sh """
+        cmake \
+          -H. \
+          -Bbuild \
+          ${cmake_options}
+    """
+    sh "cmake --build build --target irohajava -- -j${params.PARALLELISM}"
+    sh "cmake --build build --target irohapy -- -j${params.PARALLELISM}"
```

Comment:
Python tests (before launch) requires additional target to be executed, hopefully it will be fixed soon:
`cmake --build build --target python_tests`